DQM/TrackingMonitorSource/python/shortTrackResolution_cff.py

```diff
@@ -0,0 +1,65 @@
+import FWCore.ParameterSet.Config as cms
+from DQMServices.Core.DQMEDAnalyzer import DQMEDAnalyzer
+
+from RecoTracker.FinalTrackSelectors.SingleLongTrackProducer_cfi import *
+
+from RecoTracker.FinalTrackSelectors.trackerTrackHitFilter_cfi import trackerTrackHitFilter as _trackerTrackHitFilter
+TrackerTrackHitFilter = _trackerTrackHitFilter.clone(src = "SingleLongTrackProducer",
```

Comment:
I guess this clone should be called something else ( i.e
 NOT `TrackerTrackHitFilter` which is already in use)

Reply:
I changed `TrackerTrackHitFilter` to the more reasonable choice of `ShortTrackCandidates` and, to fix some of the RelVal issues, also changed `HitFilteredTracks` to `RefitedShortTracks`. Let me know if you have more suitable naming suggestions.

I am not really sure what is causing the other issues and how to solve them (see one example [here](https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-8c6f42/36649/runTheMatrix-results/24896.0_CloseByPGun_CE_E_Front_120um+2026D98/step3_CloseByPGun_CE_E_Front_120um+2026D98.log)):
```
----- Begin Fatal Exception 22-Dec-2023 08:51:25 CET-----------------------
An exception of category 'NoRecord' occurred while
   [0] Processing  Event run: 1 lumi: 1 event: 1 stream: 0
   [1] Running path 'dqmoffline_1_step'
   [2] Prefetching for module TrackerTrackHitFilter/'TrackerTrackHitFilter8'
   [3] Prefetching for EventSetup module SiStripQualityESProducer/'siStripQualityESProducer'
   [4] Calling method for EventSetup module SiStripConnectivity/'sistripconn'
   [5] While getting dependent Record from Record SiStripDetCablingRcd
Exception Message:
No "SiStripFedCablingRcd" record found in the EventSetup.

 Please add an ESSource or ESProducer that delivers such a record.
----- End Fatal Exception -------------------------------------------------
```

Any suggestions?

Reply:
>  fix some of the RelVal issues, also changed HitFilteredTracks to RefitedShortTracks.

please fix the typo => `RefittedShortTracks`.

> Any suggestions?

yes, that happens because you're trying to access phase-0 SiStrip conditions from a Phaese-2 RECO+DQM job.
Please add a suitable code modifications such that this flag 

https://github.com/cms-sw/cmssw/blob/9fca027cd9b380c6d9f4244b56f3e2c9ef2899db/RecoTracker/FinalTrackSelectors/plugins/TrackerTrackHitFilter.cc#L308

is flipped in presence of the `phase2_tracker` modifier.